model/patch_lifecycle.go

```diff
@@ -291,7 +300,7 @@ func FinalizePatch(ctx context.Context, p *patch.Patch, requester string, github
 
 	_, err = thirdparty.GetCommitEvent(githubCtx, githubOauthToken, projectRef.Owner, projectRef.Repo, p.Githash)
 	if err != nil {
-		return nil, errors.Wrap(err, "Couldn't fetch commit information")
+		grip.Warning(errors.Wrap(err, "Couldn't fetch commit information").Error())
```

Comment:
Can we log this warning with grip.WithError and grip.Message to include more information (owner/repo/githash/patch ID)? If there is a real issue this would help us with debugging